hasura/migrations/default/1670795162160_alter_table_discord_roles_circles_alter_column_role/up.sql

```diff
@@ -0,0 +1 @@
+alter table "discord"."roles_circles" rename column "role" to "server_role";
```

Comment:
Can we move this into the first migration where we create the table and remove the second `alter table` migration?

Reply:
The first migration in this PR created a different table, `"discord"."circle_api_tokens"`, so we should keep this `alter table` as that table creation happened a while ago.